### PR TITLE
Issue #559: Added a root 'layouts' directory and default README.md

### LINKED ISSUE #559: Add a /layouts directory under root for custom/contrib layouts
Do we need to have a /layouts/ folder in root? 
I'm just trying to understand Layout a bit more so I can help with docs. 
Modules can declare new layouts with hook_layout_info(). So is this the way new Layouts will be contributed, via modules, or will there be a contrib section for standalone Layouts like there is for themes and modules?

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop/backdrop-issues/issues/559.

### CODE REVIEW

File: layouts/README.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Place downloaded and custom layouts in this directory to ensure clean separation
+from Backdrop core and to facilitate safe, self-contained code updates.
+Contributed layouts from the Backdrop community may be downloaded at
+http://drupal.org/project/layouts.
```
Comment (jenlampton) [inline layouts/README.txt]: Let's not add a link to a page we know doesn't exist.  In fact, we should create these pages on backdropcms.org and just have place holder content there for now.  I'll set up https://backdropcms.org/layouts if you update this link :)

**Comment by jenlampton** [comment]
Thank you for this!  I think it will go a long way to eliminate confusion :)

**Comment by ghost** [comment]
@jenlampton There you go. Might want to fix the spelling mistake on the new layouts page though: "Layouts dive pages on your site into different columns...", should be _'divide'_ :grinning:

**Comment by jenlampton** [comment]
Fixed the spelling error, thanks!  One last tiny change, can we make any new readme files .md instead of .txt please?  Sorry I forgot to mention that last time.  
I'm not sure why that image style flood protection test is failing...  That happend to me once or twice when I hadn't rebased recently. Can you make sure you are up to date?

**Comment by ghost** [comment]
> ...can we make any new readme files .md instead of .txt please?

I think we should wait and see what's decided in https://github.com/backdrop/backdrop-issues/issues/266 first, specifically considering https://github.com/backdrop/backdrop-issues/issues/266#issuecomment-69308136...

Since all the other README files will need updating too, I think we should commit this change then change anything else as part of the other issue.

Rebased (but was already up-to-date), so just merged both commits together.

**Comment by quicksketch** [comment]
> Since all the other README files will need updating too, I think we should commit this change then change anything else as part of the other issue.

Agreed here.

Let's merge this one since it's consistent with the other README.txt files for themes and modules. We can switch them all to .md at the same time.

**Pull request merged**